### PR TITLE
tests: fix flaky env tests

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
fix, tests

**Did you add tests for your changes?**
Yes
**If relevant, did you update the documentation?**
No
**Summary**

fix flaky tests. were failing on MacOS.

**Does this PR introduce a breaking change?**
No

**Other information**
No

### CODE REVIEW

File: test/env/array/array-env.test.js [hunk 1]
```diff
@@ -17,10 +17,12 @@ describe('env array', () => {
         expect(stderr).toBeFalsy();
         expect(stdout).toBeTruthy();
 
-        const devScript = spawnSync('node', [devFile]);
-        const prodScript = spawnSync('node', [prodFile]);
+        if (isWebpack5) {
```
Comment (anshumanv) [inline test/env/array/array-env.test.js:20]: then how was it working on webpack 4 before?

Comment (snitin315) [reply test/env/array/array-env.test.js:20]: I'm still investigating the root cause :dissappointed:

Comment (anshumanv) [reply test/env/array/array-env.test.js:20]: yep investigate investigate

**Comment by evilebottnawi** [comment]
@anshumanv @snitin315 ready for merge?

**Comment by snitin315** [comment]
I think we can merge 😄.

**Comment by anshumanv** [comment]
yep lets merge and investigate

**Review by anshumanv — APPROVED** [review]

**Comment by evilebottnawi** [comment]
@anshumanv need rebase some PRs

**Comment by anshumanv** [comment]
yep wip

**Pull request merged**